app/src/main/java/com/example/aaa/pinkcalculator/MainActivity.java

```diff
@@ -2,67 +2,68 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-
 import android.view.View;
 import android.widget.Button;
 import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity implements View.OnClickListener{
-
-    private TextView fieldForFirsNumber;
-    private TextView fieldForOperation;
-    private TextView fieldForSecondNumber;
-    private TextView fieldForEqually;
-    private TextView fieldForResult;
-
-    private Button buttonFor_0;
-    private Button buttonFor_1;
-    private Button buttonFor_2;
-    private Button buttonFor_3;
-    private Button buttonFor_4;
-    private Button buttonFor_5;
-    private Button buttonFor_6;
-    private Button buttonFor_7;
-    private Button buttonFor_8;
-    private Button buttonFor_9;
-    private Button buttonFor_point;
-    private Button buttonFor_equally;
-    private Button buttonFor_plus;
-    private Button buttonFor_minus;
-    private Button buttonFor_multiplication;
-    private Button buttonFor_division;
-    private Button buttonFor_all_clean;
-
+    private static final String NULL = "0";
+    private static final String ONE = "1";
+    private static final String TWO = "2";
+    private static final String THREE = "3";
+    private static final String FOUR = "4";
+    private static final String FIVE = "5";
+    private static final String SIX = "6";
+    private static final String SEVEN = "7";
+    private static final String EIGHT = "8";
+    private static final String NINE = "9";
+    private static final String POINT = ".";
+    private static final String NULL_AND_POINT = "0.";
+    private static final String PLUS = "+";
+    private static final String MINUS = "-";
+    private static final String MULTIPLICATION = "*";
+    private static final String DIVISION = "/";
+    private static final String EQUALLY = "=";
+    private static final String EMPTY_TEXT = "";
+    private static final String MESSAGE_ERROR = "Ошибка!";
```

Comment:
Please move `MESSAGE_ERROR` and `MESSAGE_DIVISION_ON_NULL` into string resources